debug/md.py

```diff
@@ -44,3 +47,84 @@ def find_hashed_stripes(conf):
             stripes.append(s)
 
     return stripes
+
+def stripe_in_list(list_head, stripe):
+    for s in list_for_each_entry("struct stripe_head", list_head, "lru"):
+        if s == stripe:
+            return True
+
+    return False
+
+def stripe_in_inactive_list(conf, stripe):
+    NR_STRIPE_HASH_LOCKS = 8
+    for i in range(8):
```

Comment:
I think you meant `range(NR_STRIPE_HASH_LOCKS)`? Otherwise that variable is not used